src/novel_applications/payment/1-to-n_private_payment.md

```diff
@@ -0,0 +1,68 @@
+
+### Private Payments for Companies
+
+Private payments are not just essential in human societies, [learn more](./private_payment.md); they are crucial for the functioning of modern businesses. While individuals value financial privacy, companies depend on it for their survival and growth in competitive markets.
+
+Companies rely on private payments for several reasons, which include:
+
+- Maintaining competitive advantages by keeping financial strategies confidential
+- Protecting intellectual property related to research and development investments
+- Ensuring market stability by preventing unintended reactions to large transactions
+- Payment of employees, suppliers, and partners
+- And many more
+
+By maintaining privacy in their financial transactions, companies can operate more effectively in competitive business environments. This privacy does not exist in current payment systems, nor in the classical banking infrastructure, nor in current blockchains. Miden brings a solution to this problem through private transactions.
+
+## How Does Miden Enable 1-to-n Payments?
+
+Through the use of the [Miden virtual machine](https://github.com/0xPolygonMiden/miden-vm), Miden enables the consumption and generation of an arbitrary number of [notes](https://docs.polygon.technology/miden/miden-base/architecture/notes/) (artificially constrained). Each of these notes can be used to transition a user's account from a state `S` to a state `S'`.
+
+We agree on the following initial state:
+
+- There are 11 users, one is Alice
+- Alice owns `10 Ether`
+- Alice wants to send `1 Ether` to each other user
+- Alice wants to remain private
+
+The process works as follows:
+
+1. Alice transitions her state from a state `S` where she has `10 Ether` to a state `S'` where she has `0 Ether` left. Generating 10 different `Notes` with 10 different recipients, being each other user.
+
+2. Alice transfers these `Notes` containing the assets to the other users using a private arbitrary solution (On-chain encrypted notes (on the roadmap), Telegram, Signal, etc.).
+
+3. The other users each receive the `Note` and transition their individual states from `S` where they have `x Ether` to `S'` where they have `x+1 Ether`, consuming the `1 Ether` placed into the `Note` by Alice into their own states.
+
+## An Application Leveraging 1-to-n Payment on Miden
+
+Leveraging the primitives mentioned above, we can imagine a powerful 1-to-n payment app that would enable companies of all sizes to manage their payments in an elegant, private, fast, cheap, streamlined way, built on top of Miden's infrastructure.
+
+Taking inspiration from applications that could leverage 1-to-n payment like [TicketMaster](https://www.ticketmaster.com/) or [Wind](https://wind.app/developer), we can imagine a simple and modern dashboard enabling multiple payments at once:
+
+<!-- Image of imagined application -->
```

Comment:
Working on the app design

Reply:
I think it is ok. We don't need an app design here.